2020/day-17/arora-aditya/day17.py

```diff
@@ -0,0 +1,231 @@
+from typing import List, Dict, Set
+from collections import *
+from functools import lru_cache
+from math import log10
+from helper.submit.submit import *
+from utils import *
+from pprint import pprint as pp
+import numpy as np
+
+
+DAY = 17
+setup(DAY)
+lines = read_file(DAY)
+
+cubes = [
+    (-1, -1, -1),
+    (-1, -1, 0),
+    (-1, -1, 1),
+    (-1, 0, -1),
+    (-1, 0, 0),
+    (-1, 0, 1),
+    (-1, 1, -1),
+    (-1, 1, 0),
+    (-1, 1, 1),
+    (0, -1, -1),
+    (0, -1, 0),
+    (0, -1, 1),
+    (0, 0, -1),
+    (0, 0, 1),
+    (0, 1, -1),
+    (0, 1, 0),
+    (0, 1, 1),
+    (1, -1, -1),
+    (1, -1, 0),
+    (1, -1, 1),
+    (1, 0, -1),
+    (1, 0, 0),
+    (1, 0, 1),
+    (1, 1, -1),
+    (1, 1, 0),
+    (1, 1, 1),
+]
+
+quads = [
+    (-1, -1, -1, -1),
+    (-1, -1, -1, 0),
+    (-1, -1, -1, 1),
+    (-1, -1, 0, -1),
+    (-1, -1, 0, 0),
+    (-1, -1, 0, 1),
+    (-1, -1, 1, -1),
+    (-1, -1, 1, 0),
+    (-1, -1, 1, 1),
+    (-1, 0, -1, -1),
+    (-1, 0, -1, 0),
+    (-1, 0, -1, 1),
+    (-1, 0, 0, -1),
+    (-1, 0, 0, 0),
+    (-1, 0, 0, 1),
+    (-1, 0, 1, -1),
+    (-1, 0, 1, 0),
+    (-1, 0, 1, 1),
+    (-1, 1, -1, -1),
+    (-1, 1, -1, 0),
+    (-1, 1, -1, 1),
+    (-1, 1, 0, -1),
+    (-1, 1, 0, 0),
+    (-1, 1, 0, 1),
+    (-1, 1, 1, -1),
+    (-1, 1, 1, 0),
+    (-1, 1, 1, 1),
+    (0, -1, -1, -1),
+    (0, -1, -1, 0),
+    (0, -1, -1, 1),
+    (0, -1, 0, -1),
+    (0, -1, 0, 0),
+    (0, -1, 0, 1),
+    (0, -1, 1, -1),
+    (0, -1, 1, 0),
+    (0, -1, 1, 1),
+    (0, 0, -1, -1),
+    (0, 0, -1, 0),
+    (0, 0, -1, 1),
+    (0, 0, 0, -1),
+    # (0, 0, 0, 0),
```

Comment:
I memed myself so hard here smh

Reply:
LOL I didn't even know that you commented this cause I had to scroll down so far on the code file